.github/infrastructure/docker-compose-emqx.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
```

Comment:
can be removed?

Reply:
If anything make them `version: '3'`. Right?

https://docs.docker.com/compose/compose-file/compose-file-v3/#compose-and-docker-compatibility-matrix